apps/condo/domains/common/utils/addressApi.ts

```diff
@@ -45,4 +50,18 @@ export class AddressApi implements IAddressApi {
 
     private suggestionsUrl: string
     private apiToken: string
+    private defaultSearchApiParams = {
+        from_bound: { value: 'country' },
+        to_bound: { value: 'house' },
+        restrict_value: true,
+    }
 }
+
+/*
+* AddressMetaCache includes normalized addressMeta, used for getting addressMeta for address value.
+*
+* */
+type AddressValue = string
+// TODO(Dimitreee): remove any
+type AddressMeta = Record<string, any>
```

Comment:
de we already have the type for that?

Reply:
add type from https://github.com/open-condo-software/condo/pull/77